distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -81,21 +81,25 @@ org.codice.ddf.system.registry-id=
 #
 # Uncomment the properties based on the desired configuration:
 
-# For Solr External Provider:
+# For Single Solr server, internally managed:
 solr.client=HttpSolrClient
-solr.http.url=_DO_NOT_EXPAND_${org.codice.ddf.system.protocol}_DO_NOT_EXPAND_${org.codice.ddf.system.hostname}:_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}/solr
-solr.data.dir=_DO_NOT_EXPAND_${ddf.data}/solr
+solr.managed=true
```

Comment:
Setting `solr.managed` to true makes DDF star/stop Solr. This is detailed in the updated documentation.